qldbdriver/result_test.go

```diff
@@ -37,28 +37,27 @@ func TestResult(t *testing.T) {
 	// Has only one value
 	mockNextPageValues[0] = mockNextValueHolder
 
+	readIOs := int64(1)
+	writeIOs := int64(2)
+	processingTimeMilliseconds := int64(3)
+	qldbsessionTimingInformation := generateQldbsessionTimingInformation(&processingTimeMilliseconds)
+	qldbsessionConsumedIOs := generateQldbsessionIOUsage(&readIOs, &writeIOs)
+
 	result := &Result{
-		ctx:          nil,
-		communicator: nil,
-		txnID:        nil,
-		pageValues:   mockPageValues,
-		pageToken:    nil,
-		index:        0,
-		logger:       nil,
+		ctx:                 nil,
+		communicator:        nil,
+		txnID:               nil,
+		pageValues:          mockPageValues,
+		pageToken:           nil,
+		index:               0,
+		logger:              nil,
+		statementStatistics: generateStatementStatistics(new(int64), new(int64), new(int64)),
```

Comment:
Why isn't this "just": `generateStatementMetrics(0, 0, 0)`

Reply:
Updated to:
```metrics: generateMetrics(0, 0, 0)```